python_modules/dagster/dagster/core/definitions/decorators/schedule_decorator.py

```diff
@@ -195,6 +196,7 @@ def _wrapped_fn(context: ScheduleEvaluationContext) -> RunRequestIterator:
 
 
 def monthly_schedule(
```

Comment:
I think we should move this wildcard one argument down. That said, we'll probably remove these because they're pre-job/op.